TTHAnalysis/python/plotter/susy-ewkino/2lss/cuts_2lss.txt

```diff
@@ -6,37 +6,11 @@ dilep: nLepFO>=2
 pt2010: LepGood1_conePt>20 && (abs(LepGood1_pdgId)!=11 || LepGood1_conePt>25) && LepGood2_conePt>10 && (abs(LepGood2_pdgId)!=11 || LepGood2_conePt>15)
 pt25:  if3(abs(LepGood1_pdgId)!=13 && abs(LepGood2_pdgId)!=11,LepGood1_conePt>25,1)
 TT: LepGood1_isTight && LepGood2_isTight
-#exclusive: nLepTight<=2 && if3(nLepFO>2,LepGood3_mvaSUSY<-0.20+(0.20+0.50)*(abs(LepGood3_pdgId)==11),1) && nTightTauSel==0
-exclusive: nLepTight_Mini<3 && nTightTauSel==0
+#exclusive: nLepTight_Mini<3 && nTightTauSel==0
+exclusive: nLepTight_Recl<3 && nTightTauSel==0
 same-sign: (LepGood1_charge*LepGood2_charge > 0)
 met: met_pt > 60
 0bjetM: nBJetMedium25 == 0
 0and1jet: nJet40<=1
 
-# some cuts that are off by default but can be turned on with -E
```

Comment:
@nachosandres tells me that in 2lSS it is used normally _Recl, so I am leaving _Recl for the time being. In case it it not the correct choice, just let me know before approving the PR :)